scripts/realdevice/stop-realdevice-notebook-b.ps1

```
@@ -0,0 +1,56 @@
+﻿# Mind Signal Phase 18.1 dual-2pc notebook B stopper
+# DE_B 만 stop. operator PC 서비스는 별개로 stop.
+#
+# Usage:
+#   powershell -ExecutionPolicy Bypass -File .\stop-realdevice-notebook-b.ps1
+
+[CmdletBinding()]
+param(
+    [int]$DePort = 5002
+)
+
+function Stop-ByPort {
```

PSScriptAnalyzer (1.25.0)

[warning] Function 'Stop-ByPort' has verb that could change system state. Therefore, the function has to support 'ShouldProcess'.

(PSUseShouldProcessForStateChangingFunctions)